applications/matter_weather_station/README.rst

```diff
@@ -103,20 +103,21 @@ This application supports the following build types:
 * ``release`` - Release version of the application.
   You can use this version to enable only the necessary application functionalities to optimize its performance.
 
-  .. note::
-      Currently, this application supports only the ``release`` build type when `Building for the nRF7002 Wi-Fi expansion board`_.
-
-* ``factory_data`` - Release version of the application that has factory data storage enabled.
-  You can use this version to enable reading factory data necessary from a separate partition in the device non-volatile memory.
-  This way, you can read information such as product information, keys, and certificates, useful for example for Matter certification.
-  See `Generating factory data`_ to learn how to put factory data into device's storage.
-  To learn more about factory data, read the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
+.. note::
+    Currently, this application supports only the ``release`` build type when `Building for the nRF7002 Wi-Fi expansion board`_.
 
 .. note::
     `Selecting a build type`_ is optional.
     The ``debug`` build type is used by default if no build type is explicitly selected.
 
+Matter weather station build configuration overlays
+===================================================
 
+* ``overlay-factory_data`` - factory data storage support enabled.
```

Comment:
When it was a build type, it was based on the release version. Does the new overlay file work with both build types?

Reply:
Yes.